data/s3/src/main/java/org/teiid/spring/data/s3/S3Configuration.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags and
+ * the COPYRIGHT.txt file distributed with this work.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.teiid.spring.data.s3;
+
+public class S3Configuration {
+
+    private String awsAccessKey;
+    private String awsSecretKey;
+    private String bucket;
+    private String region;
```

Comment:
The region should default to US-EAST-1.

We can also consider defaulting the endpoint to aws - https://s3.amazonaws.com